packages/ui/src/utils/number-helpers.ts

```diff
@@ -62,48 +62,68 @@ export const FORMAT_ABBREVIATED_NUMBER_SUFFIX = {
   THOUSAND: "K",
 };
 
+type Threshold = "thousand" | "million" | "billion";
+const DEFAULT_THRESHOLD: Threshold = "thousand";
+const THRESHOLD_TO_NUMBER_MAP = {
+  thousand: THOUSAND,
+  million: MILLION,
+  billion: BILLION,
+};
+
+export type FormatAbbreviatedNumberOptions = {
+  /**
+   * The threshold for when to start abbreviating the number.
+   *
+   * If not provided, the number will always be abbreviated.
+   */
+  threshold?: Threshold;
+};
+
 /**
  * @param value A numeric string to format.
  * @returns The formatted numeric string and the suffix (K, M, B) to make large numbers more readable.
  */
 export const formatAbbreviatedNumber = (
   value: string,
-  {
-    minimumFractionDigits,
-    showFullThousandValue,
-  }: {
-    minimumFractionDigits?: number;
-    showFullThousandValue?: boolean;
-  } = {},
+  { threshold = DEFAULT_THRESHOLD }: FormatAbbreviatedNumberOptions = {
```

Comment:
Is the default in the destructure assignment redundant since the right hand side evaluates to the same thing?

Reply:
Oh wow, I just can't read. I thought that was some weird assignment syntax because I didn't realize it was the argument input. Woof. The plains are getting to me.